tests/fute/qvaintestcase.py

```diff
@@ -0,0 +1,184 @@
+# ADD_LICENSE_HEADER
+################################################################
+# This contains the base class QvainTestCase with some helper
+# functions for tests.
+################################################################
+import os
+from tauhka.testcase import TauhkaTestCase
+from selenium.common.exceptions import NoSuchElementException, TimeoutException
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+
+
+class QvainTestCase(TauhkaTestCase):
+    def is_frontend_running(self):
+        # This is the xpath for default error page header in nginx
+        # We do not have such xpath in our frontend
+        assert self.elem_is_not_found_xpath(
+            "/html/body/center[1]/h1"), "Frontend is not running"
+
+    def verify_that_user_is_logged_in(self):
+        self.open_usermenu()
+        usermenu_fullname = self.find_element("usermenu_fullname")
+        error_msg = "the user ({user} != {expected}) is not logged in"
+        was_user_fullname = os.environ["TEST_FULLNAME"] in usermenu_fullname.text
+        assert was_user_fullname, error_msg.format(user=usermenu_fullname.text, expected=os.environ["TEST_FULLNAME"])
+        self.close_usermenu()
+
+    def open_frontpage(self):
+        # The top does not actually get you to the frontpage
+        # appTopBarImage = self.driver.find_element_by_xpath('//*[@id="app-topbar"]/a/img')
```

Comment:
I think you fixed this so the top image should now get you to the frontpage

Reply:
true and that PR has landed already

Reply:
fixed